config

```diff
@@ -0,0 +1,26 @@
+server {
+	listen 8081;
+	host 127.0.0.1;
+	server_name free_money_near_u;
+	allow_methods GET POST DELETE;
+	autoindex on;
+	cgi_ext .py .php .pl;
```

Comment:
regarding the `cgi_ext` do we need here also .cgi? If you (@lmangall) want to have also CGI scripts which are programs (like .c/.cpp cgi files) we need a way to distinguish them from the scripts (the .php files for examples) which are interpreted, right? The .c/.cpp file would be passed as first argument of argv to execv and the second ones as second arguments being the interpreter the first one.

Reply:
Yes I already asked @dantol29 in person to add a .cgi
 
 argv[0] will be the script or program itself, as an executable, so .c or .cpp but **compiled**
 argv[1 and more] would **rather** be a file taken by our program
 
A query string will not be passed to a CGI script/program in **argv, it will be passed as a meta-variable (called environment variable once it is not in the CGI anymore but in the program).

Reply:
This means is correct not to list here (cgi_ext) the .c and .cpp extensions even if we will server pages generated through .c/.cpp scripts. The distinction on the different way to call the CGI handler will be 'hardcoded' in the Router and it will not be something that can be changed anyway in the config file or should we have also a settings for the file extensions of compiled languages files which can run a script directly (without a taking a file as argument), like you could have .c or .cpp or both or none?